### PR TITLE
added Polish translation

### PR DESCRIPTION
**Pull request by alewicki95**

### CODE REVIEW

File: translations/pl.po [hunk 1]
```diff
@@ -0,0 +1,735 @@
...
+
+#. Translators: This is a setting in the 'General' tab about including apps in the Tiling Popup
+#: tiling-assistant@leleat-on-github/src/ui/prefs.ui:56
+msgid "Include apps from all workspaces"
+msgstr "Uwzględnij aplikacje ze wszystkich przestrzeni roboczych"
+
+#. Translators: This is the header for a preference group in the 'General' tab about grouping tiled windows together
+#: tiling-assistant@leleat-on-github/src/ui/prefs.ui:71
+msgid "Tile Groups"
+msgstr "grupowanie rozmieszczanych okien"
```
Comment (pkly) [inline translations/pl.po:72]:
```suggestion
msgstr "Grupowanie rozmieszczanych okien"
```

File: translations/pl.po [hunk 2]
```diff
@@ -0,0 +1,735 @@
...
+msgstr "Włącznik aplikacji oraz wyskakujące okno rozmieszczania okien"
+
+#. Translators: This is an explanation for the 'App Switcher and Tiling Popup' setting
+#: tiling-assistant@leleat-on-github/src/ui/prefs.ui:91
+msgid ""
+"This could conflict with other App Switcher (a.k.a. alt+Tab) extensions. You "
+"may need to re-enable this or the other extension after toggling this setting"
+msgstr ""
+"Może powodować konflikt z innym przełącznikiem (np. alt+Tab) rozszerzenia."
+"Możesz go włączyć ponownie lub inne rozszerzenie po przełączeniu tego ustawienia."
```
Comment (pkly) [inline translations/pl.po:116]:
```suggestion
"Może powodować konflikt z innym rozszerzeniami zmieniającymi zmianę okien"
"(np. alt+Tab). Możliwe że będziesz potrzebować zrestartować to lub"
" inne rozszerzenie po zmienie tego ustawienia" 
```

File: translations/pl.po [hunk 3]
```diff
@@ -0,0 +1,735 @@
...
+#: tiling-assistant@leleat-on-github/src/ui/prefs.ui:230
+msgid "Window Focus"
+msgstr "Aktywne okno"
+
+#. Translators: This is an explanation for the 'Window Focus' option for the 'Dynamic Keybinding Behavior' in the 'General' tab
+#: tiling-assistant@leleat-on-github/src/ui/prefs.ui:231
+msgid ""
+"Switch focus to the tiled window in the direction of the pressed shortcut"
+msgstr ""
+"Zmień  okno na aktywne zgodnie z skrótem klawiszowym"
```
Comment (pkly) [inline translations/pl.po:200]:
```suggestion
"Zmień okno na aktywne zgodnie z skrótem klawiszowym"
```

File: translations/pl.po [hunk 4]
```diff
@@ -0,0 +1,735 @@
...
+
+#. Translators: This is the placeholder text for a search field.
+#: tiling-assistant@leleat-on-github/src/extension/layoutsManager.js:340
+msgid "Type to search..."
+msgstr "Zacznij pisać, aby wyszukać..."
+
+#. Translators: This is the text that will be displayed as the name of the user-defined tiling layout if it hasn't been given a name.
+#: tiling-assistant@leleat-on-github/src/extension/layoutsManager.js:445
+msgid "Nameless layout..."
+msgstr ""
```
Comment (pkly) [inline translations/pl.po:721]:
```suggestion
msgstr "Układ bez nazwy..."
```

**Review by pkly — COMMENTED** [review]
Checked over quickly after I noticed the translation is missing from the ext after getting a new monitor. Seems generally okay to me.

**Comment by Leleat** [comment]
Sorry, I completely forgot about this PR.

Since I don't speak Polish, I'll leave it up to you two to discuss the TL. Just ping me when it's ready to be merged.

File: translations/pl.po (hunk 1 above)
Comment (alewicki95) [reply translations/pl.po:72]: Yes, my mistake

File: translations/pl.po (hunk 2 above)
Comment (alewicki95) [reply translations/pl.po:116]: @pkly yes, Your translation is better

File: translations/pl.po (hunk 3 above)
Comment (alewicki95) [reply translations/pl.po:200]: My mistake

**Comment by alewicki95** [comment]
@Leleat translations proposed by @pkly are better and can be merged to master

**Comment by pkly** [comment]
@alewicki95 you could probably just apply my suggestions from github and then this could be merged

**Comment by Leleat** [comment]
I took the liberty and applied the changes now. Thanks for the TL, you two!

**Pull request merged**